src/ActiveJobsList.js

```diff
@@ -33,13 +34,13 @@ class ActiveJobsList extends React.Component {
                 onClick={
                   (e) => {
                     e.target.setAttribute('disabled', true)
-                    this.props.onClose(job)
+                    this.props.onClose(job, getJobDescription(job))
                   }
                 }>
                 <span aria-hidden="true">&times;</span>
               </button>
               <div>
-                <JobDescription job={ job } />
+                {getJobDescription(job)}
```

Comment:
We should call `getJobDescription` once at the top of `render` and reference the variable in both places instead.